scripts/bundle-and-gitignore-deps.js

```diff
@@ -9,7 +9,7 @@ const arb = new Arborist({ path: resolve(__dirname, '..') })
 const shouldIgnore = []
 
 arb.loadVirtual().then(tree => {
-  for (const [name, node] of tree.children.entries()) {
+  for (const [, node] of tree.children.entries()) {
```

Comment:
consider: `for (const node of tree.children.values())`

Reply:
```suggestion
  for (const node of tree.children.values()) {
```